scripts/zones/Metalworks/npcs/Ferghus.lua

```diff
@@ -1,13 +1,16 @@
 -----------------------------------
 -- Area: Metalworks
---  NPC: Ferghus
+-- NPC: Ferghus
```

Comment:
pls don't do that, triggers muh ocd.

Reply:
what is the standard here? you like 2 spaces after the "--" or whatever works to line up with what's above? I've seen many different styles in dsp... trying to adopt one and stick with it. hopefully, the right one... ;-)

Reply:
The basic principle is "avoid random unrelated changes to headers unless you are going to do them *all* that way".

I have a thing for consistency.

Reply:
yea, i hear you on that... i have a thing for "tell me how to do it a certain way and lets stick with that from now on"  :smiley: i just want to get in the habit of making everything consistent as i am doing these. especially since the quests i'm scripting are getting more and more complex.

Reply:
I've changed that to satisfy your OCD